MIGRATION_GUIDE_v3_to_v5.md

```diff
@@ -99,6 +99,14 @@ Once (or if) the user is no longer identifiable in your app (i.e. they logged ou
 
 Logging out has the affect of reverting to a “device-scoped” user, which is the new owner of the device’s push subscription.
 
+To observe changes to the onesignalId or externalId you can add a custom method to the event:
```

Comment:
I think "onesignalId" could be changed to "OneSignal Id" and "externalId" to "external Id" to avoid it being confused for variables